server/src/main/resources/db/changelog/20180110092945-cleanup-uebercerts-round4-premigration.xml

```diff
@@ -0,0 +1,100 @@
+
+    <property
+            dbms="postgresql,oracle,hsqldb"
+            name="round4_premigration_cleanup_ueber_ents"
+            value="delete from cp_entitlement where pool_id in (select id from cp_pool p where p.productname LIKE '%_ueber_product');" />
+
+    <property
+            dbms="postgresql,oracle,hsqldb"
+            name="round4_premigration_cleanup_ueber_pools"
+            value="delete from cp_pool where productname like '%_ueber_product%';" />
+
+    <property
+            dbms="postgresql,oracle,hsqldb"
+            name="round4_premigration_cleanup_ueber_content"
+            value="delete from cp_content where id in (select id from cp_content where name = 'ueber_content');" />
+
+    <property
+            dbms="postgresql,oracle,hsqldb"
+            name="round4_premigration_cleanup_ueber_products"
+            value="delete from cp_product where name like '%_ueber_product';" />
+
+    <property
+            dbms="postgresql,oracle,hsqldb"
+            name="round4_premigration_cleanup_ueber_consumers"
+            value="delete from cp_consumer where name = 'ueber_cert_consumer';" />
+
+    <!-- MySQL Queries -->
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_subs"
+            value="delete s from cp_subscription s inner join cp_product p on s.product_id = p.id where p.name LIKE '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_ent_certs"
+            value="delete ec from cp_ent_certificate ec inner join cp_entitlement e on ec.entitlement_id = e.id inner join cp_pool p on e.pool_id = p.id where p.productname like '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_ents"
+            value="delete e from cp_entitlement e inner join cp_pool p on e.pool_id = p.id where p.productname like '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_pools"
+            value="delete p from cp_pool p where p.productname like '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_content"
+            value="delete c from cp_content c
+               inner join cp_product_content pc on pc.content_id=c.id inner join cp_product p on p.id=pc.product_id where p.name LIKE '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_products"
+            value="delete p from cp_product p where p.name LIKE '%_ueber_product';" />
+
+    <property
+            dbms="mysql"
+            name="round4_premigration_cleanup_ueber_consumers"
+            value="delete c from cp_consumer c where c.name = 'ueber_cert_consumer';" />
```

Comment:
Why can't we let Liquibase do this for use?

```xml
<delete tablename="cp_consumer">
  <where>name = 'ueber_cert_consumer'</where>
</delete>
```

I tested this on Postgres and MySQL and it worked.

Postgres: 
```
DEBUG 1/11/18 2:03 PM: liquibase: db/changelog/changelog-update.xml: db/changelog/20180110092945-cleanup-uebercerts-round4-premigration.xml::20180110092945-1::vrjain: Executing EXECUTE database command: DELETE FROM public.cp_consumer WHERE name = 'ueber_cert_consumer'
```

MySQL: 
```
DEBUG 1/11/18 2:01 PM: liquibase: db/changelog/changelog-update.xml: db/changelog/20180110092945-cleanup-uebercerts-round4-premigration.xml::20180110092945-1::vrjain: Executing EXECUTE database command: DELETE FROM candlepin.cp_consumer WHERE name = 'ueber_cert_consumer'
```

Reply:
@awood I simply copied the existing changelog from 0.9.54-HOTFIX. we could re-write them though. please let me know.